src/status_im/constants.cljs

```diff
@@ -75,11 +76,11 @@
                            :UpstreamConfig {:Enabled true
                                             :URL     "https://ropsten.infura.io/z6GCTmjdP3FETEJmMBI4"}}}
    "rinkeby"     {:id     "rinkeby",
-                  :name   "Rinkeby",
+                  :name   "Rinkeby with discovery",
```

Comment:
do we want to show this for users?

Reply:
Why not? Eventually, we will have `Rinkeby` but for now, the idea is to increase awareness that Rinkeby is run in discovery mode.

Reply:
@adambabik Sounds too low level. Also there is only one Rinkeby option so it sounds superfluous. Maybe we could document this in the wiki?

Reply:
Not sure if anyone will check it there. We will get rid of it in two weeks I hope.